Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -178,6 +178,18 @@ def keg_contain?(string, keg, ignores)
           offset, match = str.split(" ", 2)
           next if linked_libraries.include? match # Don't bother reporting a string if it was found by otool
 
+          # Do not report matches to directories that do not exist.
+          dir = Pathname.new(File.dirname(match))
+          next unless dir.directory?
+
+          # Do not report matches to build dependencies.
+          begin
+            keg_name = Keg.for(dir).name
```

Comment:
Does this match cases where it's a random path inside a `Keg` rather than the actual keg?

Reply:
Yes